install.sh

```diff
@@ -0,0 +1,58 @@
+#/
+# Copyright 2015-2016 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#/
+#!/bin/bash
+#
+# use the command line interface to install pushnotifications package.
+# this script is in blue because it need blue whisk.properties.
+#
+
+#: ${WHISK_SYSTEM_AUTH:?"WHISK_SYSTEM_AUTH must be set and non-empty"}
+AUTH_KEY=`cat ../../../credentials/auth.whisk.system`
+
+SCRIPTDIR="$(dirname "$0")"
+CATALOG_HOME='../'
+source "$CATALOG_HOME/util.sh"
+
+# pushnotifications actions
+
+echo Installing pushnotifications package.
+
+createPackage pushnotifications \
+-a description "pushnotifications service" \
+-a parameters '[ {"name":"appId", "required":true}, {"name":"appSecret", "required":true}]'
+
+waitForAll
+
+install "$CATALOG_HOME/pushnotifications/webhook.js" \
```

Comment:
Don't assume the directory `$CATALOG_HOME/pushnotifications`
Have the user cd into the root of the package and then run `./install.sh <apihost> <apikey>`

Instructions in the readme will look something like

`git clone https://github.com/openwhisk/wsk-pkg-pushnotifications`
`cd wsk-pkg-pushnotifications`
`./install.sh` `192.168.1.23` `23bcnijfjjfofof`